UPGRADE.md

```diff
@@ -17,6 +17,42 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 
 ## [From 7.0.0-alpha4 to Unreleased]
 
+### [shopsys/framework]
+#### PostgreSQL upgrade:

```

Comment:
Hello @MattCzerner,

I have made a few changes to your text to enhance grammatical correctness and fixed few nonsenses:

`#### PostgreSQL upgrade:
We decided to move onto a newer version of PostgreSQL.

These steps are for migrating your data onto the newer version of PostgreSQL and are inspired by [official documentation](https://www.postgresql.org/docs/10/static/upgrading.html):

In case you are running your project natively follow [official instructions](https://www.postgresql.org/docs/10/static/upgrading.html). If you are using docker infrastructure you can follow the steps written below.

1. create a backup of your database by executing:

    `docker exec -it shopsys-framework-postgres pg_dumpall > backupfile`

1. copy new version of `docker-compose`:

    `cp docker/conf/docker-compose.yml.dist`

    *Note: select correct `docker-compose` according to your operating system*

1. stop containers and delete old data:

    `docker-compose down`

    `rm -rf <project-name>/var/postgres-data`

1. start a new docker-compose stack with the newer version of PostgreSQL by just recreating your containers:

    `docker-compose up -d --force-recreate`

1. copy backup into postgres container root folder

    `docker cp backupfile shopsys-framework-postgres:/`

1. restore your data:

    `docker exec -it shopsys-framework-postgres psql -d postgres -f backupfile``